app/domain/token_factory.rb

```diff
@@ -10,9 +12,19 @@ class TokenFactory < Dry::Struct
   def signing_key(account)
     slosilo["authn:#{account}".to_sym] || raise(NoSigningKey, account)
   end
-    
+
+  def signed_host_key(account:, username:)
+    exp_t = Time.now + Rails.application.config.conjur_config.hosts_authorization_token_ttl.to_i
+    signing_key(account).issue_jwt(sub: username, exp: exp_t)
```

Comment:
Alternatively, we can use a Rails date helper to clean this up:
```ruby
signing_key(account).issue_jwt(
  sub: username, 
  exp: Rails.application.config.conjur_config.hosts_authorization_token_ttl.to_i.seconds.from_now
)
```

Reply:
Please move this and other comments to be to the JIRA item - as was required by our managers